src/_wcs/components-template/Component-Template/new-component.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './new-component.st.css';
+
```

Comment:
I would choose a better directory name than "Component-Template" (this is what shows in the template selection dropdown). How about "First-Time-Template"? "Custom-Template"?

Reply:
After discussion with @JoMarton let's rename it to 'Example-Template' it suggests it's only an example and user can create their own custom templates. In regards of the text within it, I agree we need a better call to action there, I'll take it with Eli and update.
@YardenPorat can we change it?

Reply:
But it's a different repo, right? @mikawix (Aren't we doing this discussion in the wrong repo?)